src/lib.rs

```diff
@@ -106,7 +106,7 @@ mod tests {
     }
 
     #[test]
-    fn series_sync_and_async_no_waiting() {
+    fn series_sync_and_async_immediate_notify() {
```

Comment:
Same as above

Reply:
Resolved in new, 3rd commit in the stack.